roles/installer/tasks/23_rhcos_image_paths.yml

```diff
@@ -2,15 +2,30 @@
 - name: RHCOS image path (pre 4.8)
   block:
     - name: Get COMMIT_ID
-      shell: |
-        /usr/local/bin/{{ installer_cmd }} version | grep '^built from commit' | awk '{print $4}'
-      register: commit_id
```

Comment:
Could fail sooner:
```yaml
    - name: Get COMMIT_ID
      shell: |
        set -xeo pipefail
        /usr/local/bin/{{ installer_cmd }} version | grep '^built from commit' | awk '{print $4}'
      register: commit_id
    
```

Reply:
I agree that this approach is better now, I removed the ttp requirement, however, the test didn't visit the tasks we changed so, I can't tell if the requirement in ansible.util take an action. Can I suggest another way to parse the output? I think we can still use command without pipe and get the information we want with some ansible filters

Reply:
> he test didn't visit the tasks we changed

Correct, it is because this is only applicable for OCP 4.8 or older.  Are you consuming OCP 4.8 or anything older?

> Can I suggest another way to parse the output? I think we can still use command without pipe

Sure